cli/cmd/compose/convert.go

```diff
@@ -39,22 +43,33 @@ func convertCommand(p *projectOptions) *cobra.Command {
 	opts := convertOptions{
 		projectOptions: p,
 	}
-	convertCmd := &cobra.Command{
+	cmd := &cobra.Command{
 		Aliases: []string{"config"},
 		Use:     "convert SERVICES",
 		Short:   "Converts the compose file to platform's canonical format",
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if opts.quiet {
+				if opts.quiet {
```

Comment:
Guessing these shouldn't both be `opts.quiet`?

Reply:
not sure what I had in mond dogin this ( ဖ ͜ စ)

Reply:
Making _really_ sure it's enabled :)